projects/ngx-datatable/src/lib/types/drag-events.type.ts

```diff
@@ -0,0 +1,11 @@
+export type DragEventType = 'DRAG' | 'DRAG_END' | 'DRAG_ENTER' | 'DRAG_LEAVE' |
+'DRAG_OVER' | 'DRAG_START' | 'DROP';
```

Comment:
@chintankavathia Could you get these from an existing typescript type or even from the native one? Otherwise, at least make them lower case like this as I don't see the UPPER_CASE being used in ngx-translate.
```suggestion
export type DragEventType = 'drag' | 'drag-end' | 'drag-enter' | 'drag-leave' |
'drag-over' | 'drag-start' | 'drop';
```

Reply:
there is no typescript type and native one is used as string.

Reply:
@chintankavathia Thanks, I like that you aligned it with the naming of the native events.